packages/manager/modules/vps/src/routing.js

```diff
@@ -55,6 +70,12 @@ export default /* @ngInject */ ($stateProvider) => {
               }
               return true;
             }),
+        tabSummary: /* @ngInject */ (serviceName, VpsService) => {
+          const forceRefresh = true;
+          return VpsService.getTabSummary(serviceName, forceRefresh);
```

Comment:
I like it but I don't know if it's a good idea !

But this way, we know that the second property is about `forceRefresh`ing ! I don't know 😄

Reply:
Yes, I was haunted by a certain `myMethod(true, true, true, true)` :p

Reply:
If it were only me, all properties would be named if the parameter isn't obvious enough

for example in ReasonML:

```js
let addCoordinates = (~x, ~y) => {
  /* use x and y here */
};
/* ... */
addCoordinates(~y=6, ~x=5);
```
this way you always know what properties are for